com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -40,7 +40,7 @@ public sealed class ShaderResources
             [Reload("Shaders/Utils/CopyDepth.shader")]
             public Shader copyDepthPS;
 
-            [Reload("Shaders/Utils/ScreenSpaceShadows.shader")]
+            [Obsolete("Obsolete, this feature will be supported by 'RendererFeature' in the future")]
```

Comment:
Could update the text to say to mention the renderer feature we added that replaces this.

Reply:
Applied it :)